app/src/main/res/layout/list_item_online_user.xml

```diff
@@ -12,14 +12,11 @@
         android:layout_width="40dp"
         android:layout_height="40dp"
         android:layout_marginBottom="8dp"
```

Comment:
`@dimen/meshim_margin_small`

Reply:
Oh and now that I'm looking here I think those heights are defined in `@dimen` as well. If not they should be.